spec.html

```diff
@@ -31113,10 +31113,10 @@ <h1>Term</h1>
           <h1>Runtime Semantics: RepeatMatcher ( _m_, _min_, _max_, _greedy_, _x_, _c_, _parenIndex_, _parenCount_ )</h1>
           <p>The abstract operation RepeatMatcher takes eight parameters, a Matcher _m_, an integer _min_, an integer (or &infin;) _max_, a Boolean _greedy_, a State _x_, a Continuation _c_, an integer _parenIndex_, and an integer _parenCount_, and performs the following steps:</p>
           <emu-alg>
-            1. If _max_ is zero, return _c_(_x_).
+            1. If _max_ is ±0, return _c_(_x_).
             1. Let _d_ be an internal Continuation closure that takes one State argument _y_ and performs the following steps:
-              1. If _min_ is zero and _y_'s _endIndex_ is equal to _x_'s _endIndex_, return ~failure~.
-              1. If _min_ is zero, let _min2_ be zero; otherwise let _min2_ be _min_ - 1.
+              1. If _min_ is ±0 and _y_'s _endIndex_ is equal to _x_'s _endIndex_, return ~failure~.
+              1. If _min_ is ±0, let _min2_ be zero; otherwise let _min2_ be _min_ - 1.
```

Comment:
`_max_` and `_min_` come from a Quantifier, so can't be -0, so each of these 3 tests can be `is +0` (or `is 0` if you like).